openmetadata-ui/src/main/resources/ui/playwright/support/entity/ingestion/MySqlIngestionClass.ts

```diff
@@ -105,7 +105,12 @@ class MysqlIngestionClass extends ServiceBaseClass {
 
       await page.click('[data-testid="agents"]');
       await page.waitForSelector('[data-testid="ingestion-details-container"]');
-      await page.waitForTimeout(1000);
+
+      const metadataTab = page.locator('[data-testid="metadata-sub-tab"]');
+      if (await metadataTab.isVisible()) {
+        await metadataTab.click();
+      }
```

Comment:
by default it will be visible in both the case, collate as well as OSS, so i dont think this will help.

Reply:
may be use is_oss flag for this,

Reply:
Or may be is not checked will help?

Reply:
No, we are not showing the sub tab switch in the OSS since there is only one tab there. So this should work

Reply:
Amazing!